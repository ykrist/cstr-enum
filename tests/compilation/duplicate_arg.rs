use cstr_enum::*;

#[derive(AsCStr)]
enum Enum0 {
  #[cstr(name="egg")]
  #[cstr(name="egg")]
  A,
  B,
  C
}

#[derive(AsCStr)]
enum Enum1 {
  #[cstr(name="egg", name="foo")]
  A,
  B,
  C
}

fn main() {

}