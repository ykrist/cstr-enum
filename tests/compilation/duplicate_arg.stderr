error: duplicate named argument
 --> $DIR/duplicate_arg.rs:6:10
  |
6 |   #[cstr(name="egg")]
  |          ^^^^

error: duplicate named argument
  --> $DIR/duplicate_arg.rs:14:22
   |
14 |   #[cstr(name="egg", name="foo")]
   |                      ^^^^
